packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -9,11 +9,13 @@ const helpText = `
 Options
   --detach Start the copy without waiting for it to finish
   --attach <job-id> Attach to the running copy process to show progress
+  --skip-history Don't preserve document history on copy
```

Comment:
Isn't it the _dataset_ history that is skipped?

Reply:
That might be! I'm unsure as to how to phrase this in the best way 🙂 My thinking is that what we're "actually" preserving is the history of all the documents within the dataset right? Like creation, versions and so on. It's not the history of the dataset *itself* we're preserving. But if it makes more sense to say "dataset history" then we can change this 🙂

Reply:
No, I think document history is correct as it is. History is per document, and there's no history outside of that.

Reply:
My mistake!